tests/kubectl_test.go

```diff
@@ -232,4 +232,94 @@ var _ = Describe("[sig-compute]oc/kubectl integration", func() {
 			Expect(output).To(ContainSubstring("timestamp"))
 		})
 	})
+
+	Describe("oc/kubectl get vmi: primary interface ip validation", func() {
```

Comment:
IMO we should not have such a dedicated e2e test, it is too small scoped to worth the resources cost.
But even if the vote is to keep one, I think a single test is enough, not 3.

As I see it, we beed only one test for networking, which should cover all we expect to get as output. I.e. check the full details one will expect to see.
Edge cases can be covered by unit tests, especially if they are not critical to the system functionality.

Reply:
The e2e test was added following @AlonaKaplan recommendation.
I can minimize it to having one test for the main case of primary iface not at first position.
Unit tests cover the edge cases.